pytorch_lightning/strategies/parallel.py

```diff
@@ -136,3 +134,14 @@ def block_backward_sync(self):
     def teardown(self) -> None:
         self.cluster_environment.teardown()
         super().teardown()
+
+    @property
+    def torch_distributed_backend(self) -> str:
```

Comment:
It would be great if we could keep it to the top where all other properties belong